superset/cli.py

```diff
@@ -57,14 +60,24 @@ def runserver(debug, no_reload, address, port, timeout, workers):
             debug=True,
             use_reloader=no_reload)
     else:
-        cmd = (
-            "gunicorn "
-            "-w {workers} "
-            "--timeout {timeout} "
-            "-b {address}:{port} "
```

Comment:
No need to duplicate everything `"-b unix:{socket} " if socket else "-b {address}:{port} "` should work too